routes/plugins/error.js

```diff
@@ -12,13 +13,15 @@ exports.register = (server, options, next) => {
     const error = Boom.wrap(request.response);
     const accept = request.headers.accept || '';
 
-    if (error.status >= 500) {
-      request.server.log(['error'], error.stack);
+    if (error.output.statusCode >= 500) {
+      console.error(error.stack);
     }
 
     if (accept.indexOf('text/html') === 0) {
       // Respond with an error template
-      return reply.view(options.template || 'error', error).code(error.output.statusCode);
+      return reply
+        .view(options.template || 'error', { error, isDevelopment })
+        .code(error.output.statusCode);
```

Comment:
Apologies this should really be in a separate PR - this just adds `isDevelopment` boolean to the template data so that it can output the error stack trace if in development.